src/ibl_to_nwb/tools/ecephys/spikeglx_to_nwb.py

```diff
@@ -0,0 +1,194 @@
+from typing import Optional
```

Comment:
I'd rename the file from `spikeglx_to_nwb.py` to `ibl_recording_to_nwb.py` or something since if you're using the streaming usage, it kind of becomes decoupled from the SpikeGLX format